frontend/src/authentication/components/GuestGuard.js

```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types';
 import { Navigate } from 'react-router-dom';
-import { useAuth } from '../hooks';
+import { useAuth } from '../hooks/useAuth';
```

Comment:
I think this change is not needed. Hooks already exports useAuth

Reply:
Addressed this